apps/public-docsite/tsconfig.json

```diff
@@ -1,23 +1,18 @@
 {
+  "extends": "../../tsconfig.base.v8.json",
   "compilerOptions": {
     "outDir": "lib",
     "target": "es5",
     // ensure webpack can split async imports into separate chunks
     "module": "esnext",
     "jsx": "react",
     "declaration": true,
-    "sourceMap": true,
     "experimentalDecorators": true,
     "importHelpers": true,
-    "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node",
     "preserveConstEnums": true,
-    "noImplicitThis": true,
```

Comment:
v8 will not compile with `noImplicitThis` so removing from config. 

> Note that no `this` is used within this codebase.

All packages needs to conform to same TS compiler standards. One of many advantages using path aliases for robust and uniform code.